src/02-common-principles.md

```diff
@@ -498,6 +498,16 @@ Note that if a field name included in the data dictionary matches a column name
 then that field MUST contain a description of the corresponding column,
 using an object containing the following fields:
 
+{{ MACROS___make_metadata_table(
+   {
+      "LongName": "OPTIONAL",
+      "Description": "RECOMMENDED",
+      "Levels": "RECOMMENDED",
+      "Units": "RECOMMENDED",
+      "TermURL": "RECOMMENDED",
+   }
+) }}
+
```

Comment:
table seems to be (rendered) empty, shortcut link https://bids-specification--762.org.readthedocs.build/en/762/02-common-principles.html#tabular-files